### PR TITLE
FE-1134: Update babel to include optional-chaining

### PR DESCRIPTION
**Pull request by kchu1337**
### What Changed
 - Updated babel core and babel preset

### How To Test
 - Verify that the codebase compiles with npm start
 - Verify that there are no more instances of `({...foo}) => {...}`

### CODE REVIEW

File: src/components/pieChart/LegendItem.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ function OGLegendItem(props) {
     otherHovered && styles.otherHovered,
   );
   const breadcrumbMarkup = breadcrumb ? <ChevronLeft className={styles.BreadcrumbIcon} /> : null;
-  const expandMarkup = children && children.length ? <ChevronRight /> : null;
+  const expandMarkup = children?.length ? <ChevronRight /> : null;
```
Comment (kchu1337) [inline src/components/pieChart/LegendItem.js:21]: Testing optional chaining

File: src/components/pieChart/Legend.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const Legend = ({
   hoveredItem,
 }) => {
   const styles = useHibanaOverride(OGStyles, hibanaStyles);
-  const hoveredDataSet = hoveredItem && hoveredItem.dataSet;
+  const hoveredDataSet = hoveredItem?.dataSet;
```
Comment (kchu1337) [inline src/components/pieChart/Legend.js:19]: Testing optional chaining

File: src/components/CopyToClipboard/tests/CopyToClipboard.test.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ describe('CopyToClipboard Component', () => {
   window.setTimeout = jest.fn();
   window.clearTimeout = jest.fn();
 
-  const subject = ({ ...props }) => shallow(<CopyToClipboard {...props} />);
```
Comment (kchu1337) [inline src/components/CopyToClipboard/tests/CopyToClipboard.test.js:11]: This format breaks tests.

File: package.json [hunk 4]
```diff
@@ -123,7 +123,7 @@
     "validator": "^10.2.0"
   },
   "devDependencies": {
-    "@babel/core": "7.1.6",
+    "@babel/core": "7.10.5",
```
Comment (nicklemmon) [inline package.json:126]: I believe this update is breaking our local storybook build: `Missing class properties transform.`

**Pull request merged**